arkane/pdep.py

```diff
@@ -297,7 +297,7 @@ def execute(self, output_file, plot, file_format='pdf', print_summary=True):
                 self.plot(os.path.dirname(output_file))
             if self.sensitivity_conditions is not None:
                 logging.info('\n\nRunning sensitivity analysis...')
-                SensAnalysis(self, os.path.dirname(output_file), perturbation=self.sensitivity_perturbation)
+                SensAnalysis(deepcopy(self), os.path.dirname(output_file), perturbation=self.sensitivity_perturbation)
```

Comment:
Should this be assigned to anything? Or does it change things in place?

Reply:
It doesn't change anything in place, but it executes on initialization after the associated main pdep job is finished so it doesn't matter that it isn't assigned to anything.

Reply:
I'm confused, could you help me understand why a copy of self is needed here if it doesn't change things in place?

Reply:
So I made this commit years ago so I don't precisely remember, but SensAnalysis relies on modifying the network to do what it does so my guess is that the issue is that if you don't copy it it saves itself incorrectly.